myleaderlog.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+echo "Please note that this relies on having a cloned version of:https://github.com/DamjanOstrelic/cardano-leader-logs."
+echo "Then change the directory where this cloning is done into CARDANO_LLOG_EXEC parameter"
+echo "Also we need to have vrf.skey into the VRF_KEY_FILE parameter"
+
+
+SLOTLEADER_CONFIG="$HOME/projects/cardano-scripts/src/cscripts/slotLeaderconfig.json"
+LEDGER_FILE="/tmp/ledger.json"
+VRF_KEY_FILE="$HOME/projects/cardano-scripts/src/kaddr_run/vrf.skey"
+CARDANO_LLOG_EXEC="$HOME/projects/cardano-leader-logs/"
+cardano-cli query ledger-state --mainnet --out-file $LEDGER_FILE
```

Comment:
we don't need to dump ledger state anymore. We can use cardano-cli for querying pool stake and total active stake.

Reply:
ok. Let me change and resubmit PR. Then the argv changes will disappear.

Reply:
It would be nice if we could test it without the buffer size option, and then add those changes to this PR as well if it can be removed